tools/go.mod

```diff
@@ -4,17 +4,13 @@ go 1.15
 
 require (
 	github.com/BurntSushi/toml v0.3.1
-	github.com/buildpacks/imgutil v0.0.0-20210513150455-55e42b288ec8
+	github.com/buildpacks/imgutil v0.0.0-20210609210403-3145c7480cde
```

Comment:
Can we get rid of `tools/*` completely now that you've upgrade the `Makefile`?

Reply:
There are still useful tools in there (`tools/image/main.go`, etc.), but I don't see any reason it has to be a separate Go module anymore. I've pushed another commit that removes `tools/go.*` and reruns `go mod download` to get all of `tools/`'s deps in `/go.mod`.